src/extension/debuggingConfiguration/debugScenarioNameGenerator.ts

```diff
@@ -84,21 +86,19 @@ export class DebugScenarioNameGenerator {
         debugScenarioName: DebugScenarioName,
         debugScenarioType: DebugScenarioType,
         debugType: string,
+        useHermesEngine: boolean,
         platformType?: PlatformType | string,
     ) {
+        if (useHermesEngine) {
+            if (debugScenarioType === DebugScenarioType.AttachApp) {
+                debugScenarioName.postPlatformTypeDescription = "Hermes application";
+            } else {
+                debugScenarioName.postPlatformTypeDescription = "Hermes";
+            }
```

Comment:
Maybe it's better to use the ternary operator here?
```suggestion
            debugScenarioName.postPlatformTypeDescription = (debugScenarioType === DebugScenarioType.AttachApp) ? "Hermes application" : "Hermes";
```